src/software/test_util/test_util.h

```diff
@@ -2,6 +2,8 @@
 
 #include <gtest/gtest.h>
 
+#include <chrono>
```

Comment:
What's this used for?

Reply:
It's used to calculate time since a chrono time. I think it's needed because I removed iostream from test_util.cpp